README.md

```diff
@@ -1,3 +1,38 @@
 # Unread Messages Plugin
 
-Adds the ability to count unread messages in ChatEngine.Chats.
+Adds the ability to count unread messages in ChatEngine.Chats
```

Comment:
Same comment about keeping it chill in plugin descriptions. Can be changed to ChatEngine Chats. nbd